### PR TITLE
gruvbox-gtk-theme: unstable-2022-12-09 -> unstable-2023-05-26

### PR DESCRIPTION
**Pull request by cch000**
###### Description of changes

Updated the rev to the latest one

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Gerg-L** [comment]
Please squash and rename your commits and rename the PR

**Comment by Gerg-L** [comment]
Something like `gruvbox-gtk-theme: 2022-12-09 -> 2023-5-26`

**Comment by cch000** [comment]
Done

**Comment by Gerg-L** [comment]
Builds for me on `x86_64-linux`
LGTM

**Review by Gerg-L — APPROVED** [review]

File: pkgs/data/themes/gruvbox-gtk-theme/default.nix [hunk 1]
```diff
@@ -6,13 +6,13 @@
 }:
 stdenvNoCC.mkDerivation rec {
   pname = "gruvbox-gtk-theme";
-  version = "unstable-2022-12-09";
+  version = "unstable-2023-5-26";
```
Comment (AndersonTorres) [inline pkgs/data/themes/gruvbox-gtk-theme/default.nix:9]:
```suggestion
  version = "unstable-2023-05-26";
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/data/themes/gruvbox-gtk-theme/default.nix [hunk 2]
```diff
@@ -6,13 +6,13 @@
...
 
   src = fetchFromGitHub {
     owner = "Fausto-Korpsvart";
     repo = "Gruvbox-GTK-Theme";
-    rev = "c3172d8dcba66f4125a014d280d41e23f0b95cad";
-    sha256 = "1411mjlcj1d6kw3d3h1w9zsr0a08bzl5nddkkbv7w7lf67jy9b22";
+    rev = "c0b7fb501938241a3b6b5734f8cb1f0982edc6b4";
+    sha256 = "sha256-Y+6HuWaVkNqlYc+w5wLkS2LpKcDtpeOpdHnqBmShm5Q=";
```
Comment (AndersonTorres) [inline pkgs/data/themes/gruvbox-gtk-theme/default.nix:15]:
```suggestion
    hash = "sha256-Y+6HuWaVkNqlYc+w5wLkS2LpKcDtpeOpdHnqBmShm5Q=";
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

**Review by Gerg-L — APPROVED** [review]

**Comment by Gerg-L** [comment]
The only other thing I can see is that `rec` is dead code

**Comment by cch000** [comment]
> The only other thing I can see is that `rec` is dead code

removed it

**Review by AndersonTorres — APPROVED** [review]
LGTM, waiting ofborg to merge.

**Review by Gerg-L — APPROVED** [review]

**Pull request merged**